### PR TITLE
General updates to the training

### PR DESCRIPTION
**Pull request by rmahroua**
Hey,
I took the training and fixed little typos here and there as I saw them.
I also added a new entry in the troubleshooting section, based on a previous deployment error.
Finally, I have updated the link to the logo in the wrap-up page to make use of the new company logo.

### CODE REVIEW

**Comment by rmahroua** [comment]
Hey Jimmy, I must have missed that, sorry.
Your last instruction was to initiate the PR against the dev branch - Were the comments in the other PR?

**Comment by cigamit** [comment]
Nope, its in this one, above it show show that I started a review and added comments.

**Comment by rmahroua** [comment]
Perhaps I'm blind or something :p , but I don't see any comments here. On which tab should I click? Do I have to do something to see those comments?
Cheers

File: docs/lab_guide/wrapup.adoc [hunk 1]
```diff
@@ -16,4 +16,4 @@ What do you think? How can we help you understand Ansible Tower better?
 
 Thank you for your time and participation!
 
-image::redhat.svg[]
+image::images/new-red-hat-logo.jpg[]
```
[the remark this answers is not in the document]
Comment (rmahroua) [reply docs/lab_guide/wrapup.adoc]: You are correct,  it got pushed not so long ago, after I made the changes. I'll revert this change

File: docs/lab_guide/exercise1.adoc [hunk 2]
```diff
@@ -167,21 +167,21 @@ If you click on the Host's name or on the Edit Button image:images/at_edit.png[E
...
 
-*ansible_connection: winrm*
-
-In our example we tell Ansible to connect to the WinRM SSL Port 5986.  The nonSSL port runs on 5985 but is unecrypted.
-
 *ansible_port: 5986*
 
 We also tell Ansible to ignore the WinRM cert, since our lab doesn't have a proper certificate store setup.
 
+*ansible_connection: winrm*
```
[the remark this answers is not in the document]
Comment (rmahroua) [reply docs/lab_guide/exercise1.adoc]: I updated the file

**Pull request merged**